src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const minDiscountDays = 3;
+  const maxDiscountDays = 7;
+  const minPersDiscount = 20;
+  const maxPersDiscount = 50;
+
+  if (days < minDiscountDays) {
+    return price * days;
+  } else if (days >= maxDiscountDays) {
+    return (price * days) - maxPersDiscount;
+  } else {
+    return (price * days) - minPersDiscount;
+  }
```

Comment:
When you return the value from the function in the `if` block, the code will not reach the `else` block, so you can refactor the code to avoid nesting:

```suggestion
  if (days < minDiscountDays) {
    return price * days;
  }
  
  if (days >= maxDiscountDays) {
    return (price * days) - maxPersDiscount;
  }
  
  return (price * days) - minPersDiscount;
```